src/sts.rs

```diff
@@ -0,0 +1,269 @@
+//! The AWS STS API.
+
```

Comment:
Let's add a line in here to make it easier to find the main client class.  This is https://github.com/rusoto/rusoto/issues/519 and there are examples in https://github.com/rusoto/rusoto/pull/544 .  A gotcha is the client struct URL is case sensitive when it's deployed to the API reference page on Github pages (https://rusoto.github.io/rusoto/rusoto/).